### ISSUE TITLE
Error in no-action rule when passing number literals

### ISSUE THREAD

**Issue by richard-viney** [ember-template-lint/ember-template-lint#928]
In `ember-template-lint@1.8.2` this template:

```html
<MyComponent @value={{1}} />
```

Gives the error:

```
  -:-  error  Cannot read property 'length' of undefined  undefined
```

It's coming from the `no-action` rule introduced in 1.6, earlier versions of this package are fine, and disabling the `no-action` rule also makes the error go away. If the attribute value is changed to be a string rather than a number literal then there's also no problem, but of course the passed value is then not a number.

This may be related to #887, which appears to report a similar issue that was fixed in 1.6.2.

Thanks!

**Comment by rwjblue** [comment]
> In ember-template-lint@1.8.2 this template:

Maybe you meant 1.8.1? I released 1.8.2 roughly 7 hours after you opened this issue....

I think this was fixed by in #923 and released in v1.8.2, can you double check version and let us know?

**Comment by richard-viney** [comment]
Apologies, that must have been a typo in the patch version there. It works perfectly in the new v1.8.2. Thanks.
